rero_ils/modules/organisations/api.py

```diff
@@ -173,6 +173,12 @@ def reasons_not_to_delete(self):
             cannot_delete['links'] = links
         return cannot_delete
 
+    def is_test_organisation(self):
+        """Check if this is a test organisation."""
+        if self.code == 'cypress':
+            return False
+        return True
+
```

Comment:
It's just the opposite :-) "Cypress" is a test organisation

Reply:
+1

Reply:
When you follow suggestions without thinking.... Thanks.